public/scripts/config.js

```diff
@@ -0,0 +1,3 @@
+var config = {
+  GM_KEY: "AIzaSyDdHIB2BSaeyBVpJVNOhaPFkGlHJhuEsjw"
+};
```

Comment:
When you tackle the google maps api key issue, you may want to include this file in the gitignore file.

Other than that looks good.
OK to merge👍